src/orchestrator/orchestrator.rs

```diff
@@ -583,9 +590,13 @@ impl<'a> JobTask<'a> {
         sender: Vec<Sender<JobResult>>,
     ) -> Self {
         let bar = prep.bar.clone();
+        let max_endpoint_name_length = prep.scheduler.max_endpoint_name_length();
         bar.set_message(format!(
-            "[{} {} {}]: Booting",
+            "{:-<max_endpoint_name_length$} {:-<CONTAINER_ID_LENGTH$} {} {} {} {} Booting",
+            "",
+            "",
```

Comment:
??? (https://github.com/science-computing/butido/pull/374#discussion_r1611985925)

Reply:
Is that information (endpoint name + container ID) not available here? In that case we should perhaps use a placeholder like dashes (that also helps with keeping the output structured/parseable).

Reply:
This is the placeholder with dashes. The given string here "" gets filled up with dashes.
Looks a bit odd, but works